lisa/tools/stress_ng.py

```diff
@@ -76,12 +76,17 @@ def launch_class_async(
             sudo=sudo,
         )
 
+    def _install_required_packages(self) -> None:
+        if isinstance(self.node.os, CBLMariner):
+            self.node.os.install_packages(["glibc-devel", "kernel-headers", "binutils"])
+
     def _install_from_src(self) -> bool:
         tool_path = self.get_tool_path()
         git = self.node.tools[Git]
         git.clone(self.repo, tool_path, ref=self.branch)
         self.node.tools.get(Gcc)  # Ensure gcc is installed
```

Comment:
gcc is installed with make.

Reply:
Make does not have the function 'dependencies', but gcc is installed in _install


    def _install(self) -> bool:
        posix_os: Posix = cast(Posix, self.node.os)
        posix_os.install_packages([self, Gcc])

Reply:
That's the same. Dependencies is installed separately, and the install_packages can install them in the same group.